articles/java/sdk/get-started.md

```diff
@@ -129,7 +139,7 @@ import com.azure.resourcemanager.compute.models.KnownLinuxVirtualMachineImage;
 import com.azure.resourcemanager.compute.models.VirtualMachine;
 import com.azure.resourcemanager.compute.models.VirtualMachineSizeTypes;
 
-public class AzureApp {
+public class App {
```

Comment:
These are for `maven-archetype-quickstart`. I am not sure whether this is behavior change of maven-archetype-quickstart
https://maven.apache.org/archetypes/maven-archetype-quickstart/